plugins/modules/ios_ping.py

```diff
@@ -32,13 +22,20 @@
 version_added: 1.0.0
 author:
 - Jacob McGill (@jmcgill298)
-extends_documentation_fragment:
-- cisco.ios.ios
+- Sagar Paul (@KB-perByte)
 options:
   count:
     description:
     - Number of packets to send.
     type: int
+  afi:
```

Comment:
If we're rewriting this, can we also add the `timeout` key?

Reply:
added it, Thanks for the suggestion.